workspaces/tech-radar/plugins/tech-radar/README.md

```diff
@@ -197,7 +197,13 @@ The TS example can be found [here](src/sample.ts).
 
 ### How do I load in my own data?
 
-The `TechRadar` plugin uses the `techRadarApiRef` to get a client which implements the `TechRadarApi` interface. The default sample one is located [here](src/sample.ts). To load your own data, you'll need to provide a class that implements the `TechRadarApi` and override the `techRadarApiRef` in the `app/src/apis.ts`.
+There are two ways to load in data to the `Tech Radar` plugin; (A) implementing a custom version of the the `TechRadarApi` interface, or (B) use the default
+[API implementation](src/defaultApi.ts), which requires installing the optional tech-radar-backend. The default api will attempt to load data from the backend, and
+surface mock data if it cannot. The options are prioritized in order; if all both are implemented, option A (the custom interface) would be used.
+
+#### Option A: Implementing the TechRadarApi
```

Comment:
shall we swap the two options? The current Option B is the recomended approach we want people to use in the future, right?

Reply:
Ah, ordered this way to showcase that adding a custom API would supercede the default behavior if both were implemented, and because the custom API is the status quo. I don't think of one option as being preferred over the other, but can swap them if you feel strongly!

Reply:
I'm thinking about new people installing the plugin. What's our recommendation? Override the techradar api via code or store a json in git?

Reply:
Both options suit different use cases - json in git is less configuration effort upfront, but has less flexibility than implementing the api. Given that (and since I'm not an owner of the plugin / don't have a strong recommendation), going to keep them in order to keep the flow order consistent with which option overrides the other if both are present. Net new users can read both options and decide which works best for their use case!